dali/operators/expressions/expression_impl_cpu.h

```diff
@@ -27,6 +27,29 @@
 
 namespace dali {
 
+template <ArithmeticOp op, typename Result, typename Input0>
```

Comment:
Why not just input, in other cases we have left and right. And I don't think we will have 3 arguments arithmetic operators.